### DISCUSSION TITLE
Translation of TB profiler in french

### DISCUSSION THREAD

**Discussion by justicengom** [jodyphelan/TBProfiler#264] [Ideas]
Hi 
Trust this message will find you well,
I will love to help with a translation of TB profiler in french.
Thank you

**Comment by jodyphelan** [comment]
Hi @justicengom ,

Thank you very much for volunteering to translate it is very much appreciated. I will quickly look to see if any of the material need updating and I'll let you when everything is ready.

**Comment by justicengom** [comment]
Hi Jody
Thank you for your message and happy to volunteer.

From: Jody Phelan ***@***.***>
Date: Tuesday, 07 February 2023 at 00:52
To: jodyphelan/TBProfiler ***@***.***>
Cc: Ngom Ngom, J, Mr ***@***.*** ***@***.***>, Mention ***@***.***>
Subject: Re: [jodyphelan/TBProfiler] Translation of TB profiler in french (Discussion #264)
You don't often get email from ***@***.*** Learn why this is important<https://aka.ms/LearnAboutSenderIdentification>
CAUTION: This email originated from outside the Stellenbosch University network. Do not click links or open attachments unless you recognize the sender and know the content is safe.


Hi @justicengom<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fjusticengom&data=05%7C01%7C%7C946a29dea4a14d990a2908db0894cfda%7Ca6fa3b030a3c42588433a120dffcd348%7C0%7C0%7C638113207478835513%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=D6stql0mZ%2BCNx6ZN%2B4d6PJ5M0nlFLPk3ao5QnF2wQaw%3D&reserved=0> ,

Thank you very much for volunteering to translate it is very much appreciated. I will quickly look to see if any of the material need updating and I'll let you when everything is ready.

—
Reply to this email directly, view it on GitHub<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fjodyphelan%2FTBProfiler%2Fdiscussions%2F264%23discussioncomment-4887614&data=05%7C01%7C%7C946a29dea4a14d990a2908db0894cfda%7Ca6fa3b030a3c42588433a120dffcd348%7C0%7C0%7C638113207478835513%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=muHBsQeX89wiB0j7pS5s6ULr7XsGqGlmz2GyDBBbSak%3D&reserved=0>, or unsubscribe<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAOSTWBKVTU7T6KM6O6EJ6QLWWF6CNANCNFSM6AAAAAAUSLEU2U&data=05%7C01%7C%7C946a29dea4a14d990a2908db0894cfda%7Ca6fa3b030a3c42588433a120dffcd348%7C0%7C0%7C638113207478835513%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=d6ZXBVQAaQUJ2sRAlLUc0tay1i2lech%2FIR7GwHhWIBU%3D&reserved=0>.
You are receiving this because you were mentioned.Message ID: ***@***.***>

The integrity and confidentiality of this email are governed by these terms. Disclaimer<https://www.sun.ac.za/emaildisclaimer/default.aspx>
Die integriteit en vertroulikheid van hierdie e-pos word deur die volgende bepalings bereël. Vrywaringsklousule<https://www.sun.ac.za/emaildisclaimer/default.aspx>

**Comment by jodyphelan** [comment]
Hi @justicengom ,

Thanks again for volunteering. To start off with it would be great if you could translate the main readme page. You can fork this repository and add your text to https://github.com/jodyphelan/TBProfiler/blob/master/docs/README-FR.md. Then you can open a pull request and I'll merge it in.

**Reply by justicengom** [reply]
Okay, noted

**Comment by justicengom** [comment]
Hi @jodyphelan 
do you see my pull request?

**Reply by jodyphelan** [reply]
Hi @justicengom,

Thankyou very much for doing this. I'm fure it will be appreciated a lot by all french speakers who use the tool!
I saw you had a PR open but closed it. I'll open it again and merge.

**Reply by jodyphelan** [reply]
This has now been merged, thanks again!

**Reply by justicengom** [reply]
You are welcome !!!

**Reply by justicengom** [reply]
I am still available if there are other parts to translate.

**Reply by jodyphelan** [reply]
Great to hear. You can have a go at translating the main docs too if you have time. Info on how to do this can be found here: https://github.com/jodyphelan/TBProfiler/blob/master/docs/adding-translations.md

**Reply by justicengom** [reply]
ok Thank you

**Reply by jodyphelan** [reply]
Thanks for the RP, it is really appreciated. The translations are now live at https://jodyphelan.github.io/TBProfiler/fr/

**Reply by justicengom** [reply]
It was a pleasure. I also learnt more during the process. Thank you
